clients/rust/src/generated/types/creator.rs

```diff
@@ -0,0 +1,22 @@
+//! This code was AUTOGENERATED using the kinobi library.
+//! Please DO NOT EDIT THIS FILE, instead use visitors
+//! to add features, then rerun kinobi to update it.
+//!
+//! [https://github.com/metaplex-foundation/kinobi]
+//!
+
+use borsh::BorshDeserialize;
+use borsh::BorshSerialize;
+use solana_program::pubkey::Pubkey;
+
+#[derive(BorshSerialize, BorshDeserialize, Clone, Debug, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct Creator {
+    #[cfg_attr(
+        feature = "serde",
+        serde(with = "serde_with::As::<serde_with::DisplayFromStr>")
+    )]
+    pub address: Pubkey,
+    pub verified: bool,
+    pub share: u8,
```

Comment:
In the program crate there's a warning that this in percentage, not basis points.  Is there an easy way to add that comment here while still autogenerating the struct?  If not, no big deal.

Reply:
Fixed, the comment on the program needs to be a rust doc (`\\\`) so it goes on the IDL.